server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/AppCenterNotification.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Employees.Contracts;
+
+    using Microsoft.Extensions.Logging;
+
+    using Notifications.Contracts;
+    using Notifications.Contracts.Models;
+
+    public class AppCenterNotification : IAppCenterNotification
+    {
+        private readonly ILogger logger;
+        private readonly INotifications notifications;
+
+        public AppCenterNotification(INotifications notifications, ILogger<AppCenterNotification> logger)
+        {
+            this.notifications = notifications;
+            this.logger = logger;
+        }
+
+        async Task IAppCenterNotification.Notify(
```

Comment:
my, what happened to public and why explicit :) Anyway, I see one problem here - you pass notification template value from outside. It's 'NewBuild'. Meaning, it can be different. However, no matter what template you send, short text and long text stay the same. You probably should rename Notify -> SendNewBuildNotification(buildVersion, mobileType), and just use notification template from this class.

Reply:
Name changed and BL target